core-common/src/test/java/org/zalando/nakadi/repository/kafka/KafkaRecordDeserializerTest.java

```diff
@@ -105,6 +107,8 @@ private ObjectNode expectedJsonEvent() {
                 .put("app_hashed", "hashed-app")
                 .put("status_code", 201)
                 .put("response_time_ms", 10)
+                .put("accept_encoding", "-")
```

Comment:
why do we need `-`, it eats space. we can have null in there.

Reply:
In this particular test - it is a test. 
Overall - I am trading stability of schema over disk usage.

Reply:
👍  makes sense to keep the current values as it is!